packages/@aws-cdk/aws-synthetics/test/canary.test.ts

```diff
@@ -0,0 +1,251 @@
+import '@aws-cdk/assert/jest';
+import { arrayWith, objectLike } from '@aws-cdk/assert';
+import * as iam from '@aws-cdk/aws-iam';
+import * as s3 from '@aws-cdk/aws-s3';
+import { App, Duration, Stack } from '@aws-cdk/core';
+import * as synthetics from '../lib';
+
+let stack: Stack;
+beforeEach(() => {
+  stack = new Stack(new App(), 'canaries');
+});
+
+test('Create a basic canary', () => {
```

Comment:
you can add all of the basic properties to this test instead of testing each one individually. If there is no special logic for a property it can be added here